runtime.txt

```diff
@@ -0,0 +1 @@
+3.7
```

Comment:
Is this required?

Reply:
Good catch, this was not clearly tied to Netlify. We had to tell it to use Python 3 instead of 2, but this was only one of several ways to do that. I've moved this to an environment variable in `netlify.toml` which is probably much easier to understand.